content/en/docs/components/multi-tenancy/getting-started.md

```diff
@@ -115,7 +113,7 @@ Kubeflow {{% kf-latest-version %}} provides automatic profile creation:
   - The Kubeflow deployment process automatically creates a profile for the user
     performing the deployment. When the user access the Kubeflow central dashboard
     they see their profile in the dropdown list.
-  - The automatic profile creation can be disabled as part of the deployment by setting the registration-flow env variable to false. And an admin can manually create profiles per user or per project and add collaborators through YAML files. 
+  - The automatic profile creation can be disabled as part of the deployment by setting the registration-flow env variable to false. And an admin can manually create profiles per user or per project and add collaborators through YAML files.
    Modify the kustomize/centraldashboard/base/parama.env to set the registration variable to false
```

Comment:
Suggestion:
- Don't want to be _that user_ but adding a blank line before and after code samples would be good IMHO.

Reply:
Thank you! Is there some guideline on suggested format? Is there a tool that can automate the formatting?

I was previously using `prettier` for yaml formatting, but I can only use it for new pages. Reformatting an existing page with an auto formatter causes too much distraction for the review.